rholang/src/main/scala/coop/rchain/rholang/interpreter/ContractCall.scala

```diff
@@ -32,7 +27,7 @@ import coop.rchain.rholang.interpreter.accounting.{Cost, CostAccounting}
   * @param dispatcher the dispatcher
   */
 class ContractCall[F[_]: Concurrent: Span](
-    space: RhoISpace[F],
+    space: RhoPureSpace[F],
```

Comment:
One more thing:
`Charge for storage in system processes.`
we'll need a test for that, probably in CostAccountingSpec.

Reply:
Which ones? It seems redundant to test for each of them since it's basically the same pattern. One could also argue that testing this at all is redundant since the cases reduce to standard COMM events (which are already tested).

Reply:
A single test that guard against regressions will suffice. So a test that exemplifies that produces done by system processes are charged for. 

>One could also argue that testing this at all is redundant since the cases reduce to standard COMM events (which are already tested).

One would be wrong to do so. The point is to make sure system processes are charged-for. They weren't after all.